src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -21,86 +26,72 @@ import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
 import FieldErrorMsg from 'components/shared/FieldErrorMsg';
 import FieldGroup from 'components/shared/FieldGroup';
 import HelpText from 'components/shared/HelpText';
-import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import useSystemIntakeContacts from 'hooks/useSystemIntakeContacts';
 import GetSystemIntakeQuery from 'queries/GetSystemIntakeQuery';
 import { UpdateSystemIntakeContactDetails as UpdateSystemIntakeContactDetailsQuery } from 'queries/SystemIntakeQueries';
 import { GetSystemIntake_systemIntake as SystemIntake } from 'queries/types/GetSystemIntake';
 import {
   UpdateSystemIntakeContactDetails,
   UpdateSystemIntakeContactDetailsVariables
 } from 'queries/types/UpdateSystemIntakeContactDetails';
+import {
+  CedarContactProps,
+  ContactDetailsForm,
+  SystemIntakeContactProps,
+  SystemIntakeRoleKeys
+} from 'types/systemIntake';
 import flattenErrors from 'utils/flattenErrors';
 import SystemIntakeValidationSchema from 'validations/systemIntakeSchema';
 
 import GovernanceTeamOptions from './GovernanceTeamOptions';
 
-export type ContactDetailsForm = {
-  requester: {
-    name: string;
-    component: string;
-  };
-  businessOwner: {
-    name: string;
-    component: string;
-  };
-  productManager: {
-    name: string;
-    component: string;
-  };
-  isso: {
-    isPresent: boolean | null;
-    name: string;
-  };
-  governanceTeams: {
-    isPresent: boolean | null;
-    teams:
-      | {
-          collaborator: string;
-          key: string;
-          name: string;
-        }[]
-      | null;
-  };
-};
+import './index.scss';
 
 type ContactDetailsProps = {
   systemIntake: SystemIntake;
 };
 
 const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
-  const {
-    id,
-    requestType,
-    requester,
-    businessOwner,
-    productManager,
-    isso,
-    governanceTeams
-  } = systemIntake;
+  const { id, requestType, requester, governanceTeams } = systemIntake;
   const formikRef = useRef<FormikProps<ContactDetailsForm>>(null);
   const { t } = useTranslation('intake');
   const history = useHistory();
-  const [isReqAndBusOwnerSame, setReqAndBusOwnerSame] = useState(false);
-  const [isReqAndProductManagerSame, setReqAndProductManagerSame] = useState(
-    false
-  );
+  const { authState, oktaAuth } = useOktaAuth();
+  const [intakeRequester, setIntakeRequester] = useState<CedarContactProps>({
+    commonName: '',
+    euaUserId: '',
+    email: ''
+  });
+
+  const flags = useFlags();
+
+  const [isReqAndBusOwnerSame, setReqAndBusOwnerSame] = useState<
+    boolean | null
+  >(null);
+  const [isReqAndProductManagerSame, setReqAndProductManagerSame] = useState<
+    boolean | null
+  >(null);
```

Comment:
The checkboxes were not defaulting correctly for me.

I don't think these states should have the possibility to be null.  You should instantiate them as false.  This then solves the problem of the exhaustive depths disable in the conditional useEffect below.  Check the set and setting the state in the same hook will always cause rerenders.

```
const [isReqAndBusOwnerSame, setReqAndBusOwnerSame] = useState<boolean>(
    false
  );
  const [
    isReqAndProductManagerSame,
    setReqAndProductManagerSame
  ] = useState<boolean>(false);
```